WTWSMS/common/buildings/WtWSMS_historical_temple.txt

```diff
@@ -0,0 +1,183 @@
+temple = {
+
+	tp_hagiasophia = {  # Hagia Sophia
+		potential = {
+			FROMFROM = { title = b_hagiasophia }
+			FROM = { had_global_flag = { flag = hagia_sophia_built days = 0 } }
+		}
```

Comment:
I tried `always = no` and Hagia Sophia wasn't visible even after the "Hagia Sophia built" event. I think it should stay as it is IF it doesn't affect the performance.